pontoon/base/models.py

```diff
@@ -116,6 +116,14 @@ def stringify(self):
     class Meta:
         ordering = ['name', 'code']
 
+    def latest_activity(self, project=None):
+        """Get latest activity for locale and project if provided."""
+        translations = Translation.objects.filter(locale=self)
+        if project:
```

Comment:
You could add tests for this if statement but it's iffy whether it's necessary or not. Up to you!